Neuro/Pathfinding/PathfindingHandler.cs

```diff
@@ -2,28 +2,62 @@
 using System.Collections.Generic;
 using System.Linq;
 using Neuro.Pathfinding.DataStructures;
+using Reactor.Utilities.Extensions;
 using UnityEngine;
 
 namespace Neuro.Pathfinding;
 
 public sealed class PathfindingHandler
 {
-    private const float GRID_DENSITY = 6f; // TODO: Fine-tune individual maps to optimize performance
-    private const int GRID_BASE_WIDTH = 100;
+    // TODO: Reset these in an un-initialize function
+    internal float gridDensity = 0;
+    internal int gridBaseWidth = 0;
 
-    private const int GRID_SIZE = (int)(GRID_BASE_WIDTH * GRID_DENSITY);
-    private const int GRID_LOWER_BOUNDS = GRID_SIZE / -2;
-    private const int GRID_UPPER_BOUNDS = GRID_SIZE / 2;
+    internal int gridSize = 0;
+    private int _gridLowerBounds = 0;
+    private int _gridUpperBounds = 0;
 
-    private Node[,] grid;
+    private Node[,] _grid;
+    private GameObject _nodeVisualPointParent;
+    private GameObject _taskVisualPointParent;
+    private GameObject _pathToTask;
 
     public void Initialize()
     {
-        GenerateNodeGrid();
+        InitializeFields();
 
+        GenerateNodeGrid();
         FloodFill(ShipStatus.Instance.MeetingSpawnCenter + Vector2.down * ShipStatus.Instance.SpawnRadius);
     }
 
```

Comment:
The pathfinding grid gets cut off here
![image](https://user-images.githubusercontent.com/32238504/230753332-e4b6fb36-8e20-4c32-a436-06bcc8bd27a7.png)

Reply:
Each map has a specific size to minimize computation. Same story as https://github.com/VedalAI/neuro-amongus/pull/35#discussion_r1161196496, the game needs to be restarted.
https://github.com/VedalAI/neuro-amongus/blob/c33103fa568171399bf6d4c9246e06cf05fc2799/Neuro/Pathfinding/PathfindingHandler.cs#L12-L15